config/jetpack-cloud-development.json

```diff
@@ -26,6 +26,7 @@
 		}
 	],
 	"oauth_client_id": 68663,
+	"zendesk_presales_chat_key": "216bf91d-f10f-4f66-bf65-a0cba220cd38",
```

Comment:
Looks like you accidentally added the same Key as the one used in config/development.json, so Jetpack Cloud is showing the WordPress.com chat window

Reply:
Oh! sorry about that..  I'm working too much in a hurry.. I forgot to push my last commit.. There.  It should be showing showing Jetpack branding now..  Make sure you pull in the latest changes.

Reply:
That did the trick, looks good